Check/DoctrineMongoDb.php

```diff
@@ -19,10 +20,19 @@ public function __construct(ManagerRegistry $registry, $connectionName = null)
 
     public function check()
     {
-        $connection = $this->manager->getConnection();
-        $connection->connect();
-        
-        if ($connection->isConnected()) {
+        $connection = $this->manager->getConnection($this->connectionName);
+
+        if (\method_exists($connection, 'connect')) {
+            // Using "mongo" PHP extension
+            $connection->connect();
+
+            if ($connection->isConnected()) {
+                return new Success();
+            }
+        } else {
+            // Using "mongodb" PHP extension
+            $connection->getManager()->executeCommand('test', new Command(['ping' => 1]));
```

Comment:
this would throw an exception on a failure?

Reply:
Yep, and apparently it's automatically converter to a failure during execution:
>FAIL Doctrine Mongo Db "default" connection: Uncaught MongoDB\Driver\Exception\AuthenticationException: Authentication failed.

Reply:
Should I try/catch it to convert it to a regular Failure, like on line 39?

Reply:
yeah we automatically convert exceptions .. the difference is just that you cannot customize the failure message. i don't really think it matters much though.